### ISSUE TITLE
Notification.requestPermission throws in DDC

### ISSUE THREAD

**Issue by drewwarren** [dart-lang/sdk#31127]
# Describe your issue compiling with DDC
Notification.requestPermission throws in DDC. The same code works fine in dart2js.

# Does this error show up at build time or at run time?
run time.

# Include repro instructions. How can we try to build (or run) this code?
```
import 'dart:html';
...
var permission = await Notification.requestPermission()
...
```
```
EXCEPTION: NullError: html$.Notification._requestPermission is not a function
STACKTRACE: 
dart:html                                                                            requestPermission
```

**Comment by vsmenon** [comment]
@terrylucas - this is fixed, right?

**Comment by lejard-h** [comment]
I am still getting this error with DDC

**Comment by vsmenon** [comment]
@lejard-h - which SDK are you on?  I think @terrylucas landed a recent fix for this.

**Comment by lejard-h** [comment]
I am on the last stable release, but have not tried on dev :/

**Comment by vsmenon** [comment]
Thanks.  Unfortunately, I don't think we'll be able to cherry-pick this fix to the old stable.

**Comment by vsmenon** [comment]
@terrylucas 

I'm marking this closed as I think it's fixed in 2.0 branches.
